go.mod

```diff
@@ -126,11 +130,10 @@ require (
 	github.com/shirou/gopsutil v2.19.11+incompatible
 	github.com/spf13/cobra v0.0.3
 	github.com/spf13/pflag v1.0.3
-	github.com/stretchr/objx v0.1.2-0.20180702103455-b8b73a35e983 // indirect
 	github.com/stretchr/testify v1.4.0
 	github.com/tsg/go-daemon v0.0.0-20200207173439-e704b93fd89b
 	github.com/tsg/gopacket v0.0.0-20190320122513-dd3d0e41124a
-	github.com/urso/magetools v0.0.0-20200106130147-61080ed7b22b // indirect
```

Comment:
@kvch I will need your help on theses two dependenices (objx, magetools), I can't explain why they get removed.
I've tested with the master brancher and I do not have this issue.

Reply:
as recommended by @urso I've ran `mage vendor`